### ISSUE TITLE
piKVM discovered continuously even after configured

### ISSUE THREAD

**Issue by bmorgenthaler** [adamoutler/pikvm-homeassistant-integration#5]
Core: 2024.6.3
Supervisor: 2024.06.0
Frontend: 20240610.1

piKVM was discovered on my network and configured and the over subsequent refreshes and restarts the same device is discovered again and asking to be configured. The hostname and IP of the discovered piKVM is the same as the currently configured one.

**Comment by adamoutler** [comment]
This is a known issue and it is due to the fact that I refuse to try to brute force existing data to detect if we've already connected to this device previously.  Theoretically this technique could be used to capture your device password.   I have submitted a PR to PiKVM to implement Avahi ZeroConf and will be able to detect devices automatically this way.

**Comment by adamoutler** [comment]
Yes IP is possible but MAC is not. However, I've submitted a PR and discussed with the PiKVM dev about implementing ZeroConf. Once that's been implemented, I plan to rewrite the entire config flow and ditch DHCP config.  With ZeroConf you can expect a much better flow.

**Comment by adamoutler** [comment]
The Hostname of PiKVM is always `pikvm` unless it's been changed so that's not a good detection method.

**Comment by adamoutler** [comment]
As a workaround you can change the PiKVM name to something other than `pikvm*`.  Eg.  `mypikvm`. This will cause DHCP to stop detecting the device. Or you can set your username/password to default so it will auto-add the device.

**Comment by adamoutler** [comment]
The Pre-release version is using ZeroConf. This fixes the problem.  Check release notes on GitHub.
